pkg/bootstrap/docker/openshift/admin.go

```diff
@@ -157,6 +157,7 @@ func (h *Helper) InstallRouter(kubeClient kclientset.Interface, f *clientcmd.Fac
 		HostNetwork:        !portForwarding,
 		HostPorts:          true,
 		ServiceAccount:     "router",
+		MaxConnections:     2000,
```

Comment:
I thought about this more last night and I think this is the wrong approach.  I think we make a maxconns of 0 mean 'use the default'.  It makes no sense for 0 to be set ever.  So let's not change this file, but handle it down below in router.go.